deps/wrapper.cc

```diff
@@ -104,6 +121,26 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
 
     mod.method("BuildJulia", &FunctionDescriptorBuilder::BuildJulia);
     mod.method("ToString", &ToString);
+
+    // class Buffer
+    // https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/common/buffer.h
+    mod.add_type<Buffer>("Buffer")
+        .method("Data", &Buffer::Data)
+        .method("Size", &Buffer::Size)
+        .method("OwnsData", &Buffer::OwnsData)
+        .method("IsPlasmaBuffer", &Buffer::IsPlasmaBuffer);
+    mod.add_type<LocalMemoryBuffer>("LocalMemoryBuffer", jlcxx::julia_base_type<Buffer>());
+    mod.method("LocalMemoryBuffer", [] (uint8_t *data, size_t size, bool copy_data = false) {
+        return std::make_shared<LocalMemoryBuffer>(data, size, copy_data);
+    });
```

Comment:
just to make sure I'm understanding what this is doing:

first line wraps the ray LocalMemoryBuffer type define dhere: https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/common/buffer.h#L56

second adds a julia method call `LocalMemoryBuffer` that returns a shared pointer to an instance of the local memory buffer, forwarding the data/size/copy arguments to the _real_ constructor (which we're not exposing).  is that right?

```suggestion
    // class LocalMemoryBuffer
    // https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/common/buffer.h#L56
    mod.add_type<LocalMemoryBuffer>("LocalMemoryBuffer", jlcxx::julia_base_type<Buffer>());
    mod.method("LocalMemoryBuffer", [] (uint8_t *data, size_t size, bool copy_data = false) {
        return std::make_shared<LocalMemoryBuffer>(data, size, copy_data);
    });
```